lakefile.lean

```diff
@@ -1,11 +1,11 @@
 import Lake
 open Lake DSL
 
-package iris where
+package Iris where
```

Comment:
```suggestion
package iris where
```
Convention for package names uses lowercase, although you are right that lean_libs should be pascal case.

Reply:
Thanks for the explanation. I updated the MR.